utilities/src/main/java/io/syndesis/qe/utils/BoxUtils.java

```diff
@@ -0,0 +1,184 @@
+        return ((Collection<?>)new BoxFolder(this.boxConnection, SYNDESIS_BOX_DIRECTORY_ID)).size();
+    }
+
+    public BoxFile getFile(String name) {
+        for (BoxItem.Info info : new BoxFolder(this.getBoxConnection(), SYNDESIS_BOX_DIRECTORY_ID)) {
+            if (name.equals(info.getName())) {
+                return new BoxFile(this.getBoxConnection(), info.getID());
+            }
+        }
+        return null;
+    }
+
+    /**
+     * From org.apache.camel.component.box.internal.BoxConnectionHelper
+     * @return box access token
+     */
+    private String getAccessToken() throws Exception {
+        final String csrfToken = String.valueOf(new SecureRandom().nextLong());
+        final String url = OAUTH_URL + boxAccount.getProperty("clientId")+ "&state=" + csrfToken;
+        final Connection.Response loginPageResponse = Jsoup.connect(url).method(Connection.Method.GET).execute();
+        Document loginPage = loginPageResponse.parse();
+
+        validatePage(loginPage);
+
+        final FormElement loginForm = (FormElement) loginPage.select("form[name=login_form]").first();
+
+        final Element loginField = loginForm.select("input[name=login]").first();
+        loginField.val(boxAccount.getProperty("userName"));
+
+        final Element passwordField = loginForm.select("input[name=password]").first();
+        passwordField.val(boxAccount.getProperty("userPassword"));
+
+        final Map<String, String> cookies = new HashMap<>(loginPageResponse.cookies());
+        Connection.Response response = loginForm.submit()
+                .cookies(cookies)
+                .execute();
+        cookies.putAll(response.cookies());
+        final Document consentPage = response.parse();
+
+        validatePage(consentPage);
+
+        final FormElement consentForm = (FormElement) consentPage.select("form[name=consent_form]").first();
+
+        consentForm.elements().removeIf(e -> e.attr("name").equals("consent_reject"));
+        //parse request_token from javascript from head, it is the first script in the header
+        final String requestTokenScript = consentPage.select("script").first().html();
+        final String requestToken = StringUtils.substringBetween(requestTokenScript, "'", "'");
+        if (requestToken == null) {
+            fail("Coudln't parse request token from " + requestTokenScript);
+        }
+        response = consentForm.submit()
+                    .data("request_token", requestToken)
+                    .followRedirects(false)
+                    .cookies(cookies)
+                    .execute();
+        final String location = response.header("Location");
+
+        final String code = StringUtils.substringAfterLast(location, "code=");
+
+        if (code == null) {
+            fail("Unable to get code from " + location);
+        }
+
+        return code;
+    }
+
+    /**
+     * From org.apache.camel.component.box.internal.BoxConnectionHelper
+     * @param page html page
```

Comment:
there is usually one more empty line in between comment and stuff like `@param` arguments